connexion/apis/abstract.py

```diff
@@ -238,22 +242,195 @@ def get_request(self, *args, **kwargs):
     @abc.abstractmethod
     def get_response(self, response, mimetype=None, request=None):
         """
-        This method converts the ConnexionResponse to a user framework response.
+        This method converts a handler response to a framework response.
+        This method should just retrieve response from handler then call `cls._get_response`.
+        It is mainly here to handle AioHttp async handler.
         :param response: A response to cast.
         :param mimetype: The response mimetype.
         :param request: The request associated with this response (the user framework request).
 
-        :type response: ConnexionResponse
+        :type response: Framework Response
```

Comment:
I think this is supposed to be an actual type/class.

Reply:
I cleaned up the type hint comments a bit.